src/read_agent_logging.in

```diff
@@ -19,23 +19,53 @@ URL="${1:-http://localhost:24231/metrics}"
 # Default to 10s if unset, as a fail-safe mechanism.
 INTERVAL=${COLLECTD_INTERVAL:-10}
 
+# 1. If the Logging Agent uses REST.
 # The part of the input that sed looks at looks like:
-#   stackdriver_successful_requests_count{grpc="false"} 255
-#   stackdriver_failed_requests_count{grpc="false",code="400"} 14
+#   stackdriver_successful_requests_count{grpc="false",code="200"} 255
+#   stackdriver_failed_requests_count{grpc="false",code="401"} 25
```

Comment:
Just for consistency, gRPC error code 3 is HTTP 400 (invalid argument). HTTP 401 translates to gRPC error 16 (unauthorized), and HTTP 403 translates to gRPC error code 7 (permission denied). I'm fine having each of them here as examples. You can also include HTTP 429 (quota error), which translates to gRPC error 8.

Reply:
Changed.